Project.toml

```diff
@@ -7,9 +7,12 @@ version = "0.2.2"
 AWS = "fbe9abb3-538b-5e4e-ba9e-bc94f4f92ebc"
 AWSS3 = "1c724243-ef5b-51ab-93f4-b0a88ac62a95"
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
+Elfutils_jll = "ab5a07f8-06af-567f-a878-e8bb879eba5a"
 GDB_jll = "a8b33d9f-0f8b-5197-9986-f85cbaa50c6c"
+Git_jll = "f8c6e375-362e-5223-8a59-34ff63f689eb"
```

Comment:
I recommend using [`Git.jl`](https://github.com/JuliaVersionControl/Git.jl) instead of `Git_jll.jl` directly.  There are a lot of things which won't work unless you set some environment variables, which `Git.jl` does for you.